canopy/app/src/components/navigation/SideNav.scss

```diff
@@ -96,3 +103,37 @@ $total-nav-width: map-get($nav-tab-spacings, 'padding') * 2 +
     background: $color-canopy-light;
   }
 }
+
+.profile-tab {
```

Comment:
This is fine for this PR, but we are should probably move this layout styling up to the design system.

Reply:
There's more that should be lifted so this is okay for now.